saleor/payment/models.py

```diff
@@ -118,6 +120,21 @@ def get_last_transaction(self):
     def get_total(self):
         return Money(self.total, self.currency)
 
+    def get_covered_amount(self):
+        """Return an amount that is covered by this payment (but not necessarily captured).
+
+        Partially refunded payments are included in the covered amount.
+        """
+        if self.charge_status == ChargeStatus.PARTIALLY_REFUNDED:
```

Comment:
Wouldn't be easier:
```
if charge_status == AUTHORIZED:
    return self.total
return self.captured_amount
```
Also, we have method `get_authorized_amount`, shouldn't we use it here?

Reply:
Re: `get_authorized_amount`. It's used once in conjunction with `get_last_payment` and my goal would be to remove both of them. I don't like `get_authorized_amount` as it refers to transactions which raises doubt whether `Payment.charge_status` and `Payment.amount` are correct and we should never allow them to be incorrect. 

Would you prefer:
```
if self.charge_status in {
    ChargeStatus.FULLY_CHARGED,
    ChargeStatus.PARTIALLY_CHARGED,
    ChargeStatus.PARTIALLY_REFUNDED
}:
    return self.captured_amount
if self.charge_status == ChargeStatus.AUTHORIZED:
    return self.total
return Decimal("0")
```
? I'm assuming that we want to return 0 for PENDING but we might want to check it when end-to-end testing

Reply:
`saleor.payment.utils.gateway_postprocess` updates the capture_amount. afair It is done only in this place, so I believe we could relay the status &/OR capture_amount.